cheroot/test/test_server.py

```diff
@@ -135,7 +135,7 @@ def raise_keyboard_interrupt(*args):
     serve_thread.start()
 
     # The thread should exit right away due to the interrupt.
-    serve_thread.join(0.5)
+    serve_thread.join(1.5)
```

Comment:
i am not sure this is needed - but saw this timeout at least once without it.

Reply:
do you have any logs of what the traceback was when this happens?

Reply:
Is this change in timeout related to the PR or incidental? If it's incidental, I'd rather see a separate issue describing the aberrant behavior (traceback) and a separate PR to work around or correct it.

Reply:
I've rebased the changes without 7c07fe8b. We can apply that later if needed.